site/content/en/reference/fn/export/_index.md

```diff
@@ -31,7 +31,7 @@ kpt fn export github-actions DIR/ --fn-path FUNCTIONS_DIR/
 kpt fn export ORCHESTRATOR DIR/ [--fn-path FUNCTIONS_DIR/] [--output OUTPUT_FILENAME]
 
 ORCHESTRATOR:
-  `github-actions` is currently the only supported workflow orchestrator.
+  Supported orchestrators are github-actions, and cloud-build.
```

Comment:
Supported orchestrators: GitHub Action, Cloud Build.

Reply:
Here they should be identical to the argument format. So I suggest not changing them.